src/me/EtienneDx/RealEstate/Messages.java

```diff
@@ -0,0 +1,84 @@
+package me.EtienneDx.RealEstate;
+
+import me.EtienneDx.AnnotationConfig.AnnotationConfig;
+import me.EtienneDx.AnnotationConfig.ConfigField;
+import me.EtienneDx.AnnotationConfig.ConfigFile;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.PluginDescriptionFile;
+
+import java.util.Arrays;
+import java.util.List;
+
+@ConfigFile(header = "Use a YAML editor like NotepadPlusPlus to edit this file.  \nAfter editing, back up your changes before reloading the server in case you made a syntax error.  \nUse dollar signs ($) for formatting codes, which are documented here: http://minecraft.gamepedia.com/Formatting_codes")
+public class Messages extends AnnotationConfig
+{
+    public PluginDescriptionFile pdf;
+
+    @ConfigField(name="RealEstate.Messages.NoTransactionFound")
+    public String msgNoTransactionFound = "$cNo transaction found at your location!";
+
+    @ConfigField(name="RealEstate.Messages.PageMustBePositive")
+    public String msgPageMustBePositive = "$cPage must be a positive option";
+
+    @ConfigField(name="RealEstate.Messages.PageNotExists")
+    public String msgPageNotExists = "$cThis page does not exist!";
+
+    @ConfigField(name="RealEstate.Messages.RenewRentNow", comment = "0: enabled/disabled; 1: type of claim")
+    public String msgRenewRentNow = "$bAutomatic renew is now $a{0} $bfor this {1}";
+
+    @ConfigField(name="RealEstate.Messages.RenewRentCurrently", comment = "0: enabled/disabled; 1: type of claim")
+    public String msgRenewRentCurrently = "$bAutomatic renew is currently $a{0} $bfor this {1}";
```

Comment:
I think we can remove the "Messages" from the path since it's redundant, and adds a tab everywhere for no particualr reason

```suggestion
    @ConfigField(name="RealEstate.NoTransactionFound")
    public String msgNoTransactionFound = "$cNo transaction found at your location!";

    @ConfigField(name="RealEstate.PageMustBePositive")
    public String msgPageMustBePositive = "$cPage must be a positive option";

    @ConfigField(name="RealEstate.PageNotExists")
    public String msgPageNotExists = "$cThis page does not exist!";

    @ConfigField(name="RealEstate.RenewRentNow", comment = "0: enabled/disabled; 1: type of claim")
    public String msgRenewRentNow = "$bAutomatic renew is now $a{0} $bfor this {1}";

    @ConfigField(name="RealEstate.RenewRentCurrently", comment = "0: enabled/disabled; 1: type of claim")
    public String msgRenewRentCurrently = "$bAutomatic renew is currently $a{0} $bfor this {1}";
```